Webpack/config/webpack.common.js

```diff
@@ -0,0 +1,41 @@
+const webpack = require('webpack');
+const path = require('path');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+module.exports = {
+	entry: ['whatwg-fetch', './src/app.js'],
+	output: {
+		path: './dist',
+		filename: 'app.bundle.js'
+	},
+	module: {
+		loaders: [{
+			test: /\.js$/,
+			exclude: /node_modules/,
+			loader: 'babel-loader'
+		},
+		{ 
+			test: /\.styl$/, 
+			exclude: /node_modules/,
+			//loader: 'style-loader!css-loader!stylus-loader' 
+			loader: ExtractTextPlugin.extract('style-loader', 'css-loader!stylus-loader')
+		},
+		{
+			test: /\.png$/,
```

Comment:
Haven't found example of usage this loader? No png picture imported in application . For example, it could be any logo.